packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/column-splitter/column-splitter.component.html

```diff
@@ -0,0 +1,13 @@
+﻿<div [ngClass]="['row', 'component', 'column-splitter', columnSplitterStyles]" [attr.id]="id || null">
+  <div *ngFor="let ph of enabledPlaceholders; let index = index">
+    <div [ngClass]="[columnWidths[+ph - 1], columnStyles[+ph - 1] || ''].join(' ').trim()">
```

Comment:
let's compute that inside of "component class" instead, it's too much to keep in markup